### DISCUSSION TITLE
zephyr project import with NXP MCUXpresso for VS code

### DISCUSSION THREAD

**Discussion by asteriskSF** [zephyrproject-rtos/zephyr#70524] [Q&A]
Sorry this question is very NXP tool specific, but NXP customer support won't answer any questions involving zephyr unless they are posted here.

I have setup an example application with the app outside of the zephyr sdk in an repository with its own west manifest. 

the directory structure:
top
top\tree\app            -- application root
top\tree\app\build  -- build directory for a specific board
top\zephyr
top\modules\hal\nxp
etc

I am importing the project but it doesn't seem to be aware of the build directory.  
The MCUXpreso for VS Code extension lists Build Configurations but doesn't actually show any of the  build directories.
When I'm viewing code, VS Code is not aware of the values set through Kconfig in the specific build.  It shows all conditional code as disabled.

Is there a way to configure the VS Code to understand the build directories and pull in the correct configuration?

**Comment by DerekSnell** [comment]
Hi @asteriskSF ,
Yes, this Zephyr repo is not the best place for questions specific to NXP's MCUXpresso extension for VS Code.  But we do have this [GitHub repo for the VS Code extension](https://github.com/nxp-mcuxpresso/vscode-for-mcux).  For any questions specific to that tool, you can create an [Issue ](https://github.com/nxp-mcuxpresso/vscode-for-mcux/issues)in that repo.

And I know you are already aware of that repo now, and already created this issue on the same topic at [syntax highlighting does not have zephyr/.config](https://github.com/nxp-mcuxpresso/vscode-for-mcux/issues/39).  We can use that issue for any further discussion on this topic.  Therefore, I am closing this discussion.

Thank you
